crates/storage/provider/src/providers/static_file/manager.rs

```diff
@@ -1320,6 +1341,23 @@ impl StatsReader for StaticFileProvider {
     }
 }
 
+/// Environment used when opening a static file provider. RO/RW.
+#[derive(Debug, Default)]
+pub enum StaticFileEnv {
```

Comment:
imo this name doesn't make much sense because there's no such thing as `Environment` in static files, right?

so perhaps we rename this to access?